sdks/java/io/jms/src/main/java/org/apache/beam/sdk/io/jms/JmsIO.java

```diff
@@ -225,6 +225,8 @@ public abstract static class Read<T> extends PTransform<PBegin, PCollection<T>>
 
     abstract Duration getCloseTimeout();
 
+    abstract @Nullable Duration getReceiveTimeout();
```

Comment:
I am thinking about making the default a non-null value (e.g. 100 ms) so user does not need to configure this in every pipeline. However on the other hand this would more-or-less change the default behavior if existing user assumes low letency. @damondouglas what do you think?

Reply:
@Abacn My vote is to leave it as you have it.

Reply:
sounds good